Control/public/common/detectorHeader.js

```diff
@@ -30,7 +31,9 @@ const detectorHeader = (model) => {
         h('.ph2', `Detector View: ${selected}`),
         h('a.f6.actionable-icon', {
           onclick: () => model.resetDetectorView('')
```

Comment:
You can add some padding to the main edit icon rather than a new element with 4px 
```suggestion
        h('a.f6.actionable-icon.ph2', {
          onclick: () => model.resetDetectorView('')
```